integration-test/src/test/scala/com/cloudera/livy/test/InteractiveIT.scala

```diff
@@ -48,9 +48,9 @@ class InteractiveIT extends BaseIntegrationTestSuite with BeforeAndAfter {
     dumpLogOnFailure(sessionId) {
       matchResult("1+1", "res0: Int = 2")
       matchResult("""sc.getConf.get("spark.executor.instances")""", "res1: String = 1")
-      matchResult("sqlContext", startsWith("res2: org.apache.spark.sql.hive.HiveContext"))
       matchResult("val sql = new org.apache.spark.sql.SQLContext(sc)",
-        startsWith("sql: org.apache.spark.sql.SQLContext = org.apache.spark.sql.SQLContext"))
+        ".*" + Pattern.quote(
```

Comment:
I think we should test with SparkSession here if the test's running in Spark 2.0. If this's not trivial, feel free to create a JIRA and do it later.